suites/apis/dbgapTest.js

```diff
@@ -250,6 +285,73 @@ Scenario('dbGaP Sync: created signed urls (from s3 and gs) to download, try crea
       + 'project phs000178, even though they have read access.');
 }).retry(1);
 
+Scenario('dbGaP Sync: Cascading Auth - create signed urls from s3 and gs to download, @dbgapSyncing @reqGoogle',
+    async ({I, fence, users}) => {
+        console.log('Use mainAcct to create s3 signed URL for file phs001194');
+        const signedUrls3Phs001194Res = await fence.do.createSignedUrl(
+            indexed_files.parentPhs001194File.did, ['protocol=s3'],
+            users.mainAcct.accessTokenHeader,
+        );
+
+        console.log('Use mainAcct to create gs signed URL for file phs001194');
+        const signedUrlgsPhs001194Res = await fence.do.createSignedUrl(
+            indexed_files.parentPhs001194File.did, ['protocol=gs'],
+            users.mainAcct.accessTokenHeader,
+        );
+        const signedUrls3phs001194Res = await fence.do.createSignedUrl(
+            indexed_files.parentPhs001194File.did, ['protocol=s3'],
+            users.mainAcct.accessTokenHeader,
+        );
+
+        const phs001194s3FileContents = await fence.do.getFileFromSignedUrlRes(
+            signedUrls3phs001194Res
+        );
+        const phs001194gsFileContents = await fence.do.getFileFromSignedUrlRes(
+            signedUrlgsPhs001194Res
+        );
+
+        console.log('Use mainAcct to create s3 signed URL for file phs000571');
+        const signedUrls3Phs000571Res = await fence.do.createSignedUrl(
+            indexed_files.childPhs000571File.did, ['protocol=s3'],
+            users.mainAcct.accessTokenHeader,
+        );
+
+        console.log('Use mainAcct to create gs signed URL for file phs000571');
+        const signedUrlgsPhs000571Res = await fence.do.createSignedUrl(
+            indexed_files.childPhs000571File.did, ['protocol=gs'],
+            users.mainAcct.accessTokenHeader,
+        );
+        const signedUrls3phs000571Res = await fence.do.createSignedUrl(
+            indexed_files.childPhs000571File.did, ['protocol=s3'],
+            users.mainAcct.accessTokenHeader,
+        );
```

Comment:
similarly here , created twice?

Reply:
Because I made a mistake 😅 thanks for catching that. I've removed the duplicate calls.